src/reducers/flow.reducer.js

```diff
@@ -58,15 +58,15 @@ export const reducer = (state, action) => {
 			);
 		} catch (error) {
 			invariant(
-				true,
+				false,
```

Comment:
ok remember why i did this, if new error is thrown it prevent return a default state breaking the app